### ISSUE TITLE
Documents gives 404

### ISSUE THREAD

**Issue by EmreSURK** [insinfo/galileo#9]
The links (all of them) in the README give 404.

**Comment by insinfo** [comment]
Yes the links are broken, I haven't had much time to solve this, but as soon as I have time I will update the README with some simple examples to create a server side dart application with Galileo, as I use it in production currently in my projects.
